accounts/models.py

```diff
@@ -7,11 +7,19 @@
 class User(AbstractUser):
     username = None
     email = models.EmailField(unique=True)
+    firebase_uid = models.CharField(max_length=128)
 
     USERNAME_FIELD = "email"
     REQUIRED_FIELDS = []
 
     objects = UserManager()
 
+    class Meta:
+        indexes = [models.Index(fields=["firebase_uid"])]
```

Comment:
What does this line do?

Reply:
It creates a database index for the column `firebase_uid`. It effectively lets us query by `firebase_uid` faster at the cost of disk space